cryptoinvestor/views/investments.py

```diff
@@ -0,0 +1,34 @@
+from cryptoinvestor.views import BaseView
+
+
+class InvestmentsListView(BaseView):
+    def get_template_name(self):
+        return 'investments/investments.html'
+
+    def get_objects(self):
+
+        context = {}
+        investments = self.app.user.account.investments
+
+        base = self.app.local_currency
+
+        for key in investments:
+            asset = self.app.assets.get(key)
+            rate = asset.rates.get(base, []).get('rate', 'N/A')
+
+            investments[key]['current_price'] = rate
+            investments[key]['profit'] = self.get_profit(investments[key]['price'] - rate)
+
+        context.update({
+            'investments': investments
+        })
+
+        return context
+
+    def get_profit(self, value):
```

Comment:
This should probably be resolved in a template. Meaning you should just send the profit to the template and the template should decide which icon to display.